build.gradle

```diff
@@ -13,8 +13,8 @@ group = project.maven_group
 dependencies {
 	//to change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}+build.+:v2"
-	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}+build.+"
+	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
```

Comment:
No. We don't use Minecraft as a dependency, so we don't need to keep track of the build number.

Reply:
The new loom version doesn't seem to like that format though. It only works if I specify the build number.
If I revert to the old format (`mappings "net.fabricmc:yarn:${project.minecraft_version}+build.+:v2"`) the build fails:
```
> Configure project :
Fabric Loom: 0.7.34

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'Mod-ARR-Checker'.
> Failed to notify project evaluation listener.
   > Failed to provide net.fabricmc:yarn:1.15.2+build.+ : java.lang.RuntimeException: Minecraft Version (1.15.2) does not match yarn's minecraft version (1.15.2+build.)
   > java.lang.NullPointerException (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 513ms

```

Reply:
Fully update Loom to 0.9-SNAPSHOT and see what happens.
If that doesn't work, just inline the latest build and leave it.

Reply:
0.9-SNAPSHOT requires Java 16 and Gradle 7. Should I upgrade them as well?

Reply:
(And upgrade to MC 1.17)

Reply:
Obviously don't upgrade to MC 1.17. The rest is fine, just for Gradle. modmuss suggested it.
https://github.com/FabricMC/fabric-example-mod/pull/113#issuecomment-917430620

Reply:
Yep, was a bit confused for a bit due to the error messages loom 0.9 put out.